### DISCUSSION TITLE
Uncaught RangeError: Maximum call stack size exceeded. with AlpineJs3

### DISCUSSION THREAD

**Discussion by elfeffe** [livewire/livewire#3321] [7. Bugs (archived)] [answered]
I'm having that error, I just switched to AlpineJS 3.
Is anyone else having the same error?

**Comment by calebporzio** [comment]
Can you provide more info on this error? Ideally reproduce it with the most minimal code possible and post that?

**Comment by elfeffe** [comment, marked as answer]
Sorry @calebporzio, this is not related to LW, I just found that it was Laravel-Views but it is fixed in their latest version.
False alarm.

**Reply by christophrumpel** [reply]
Hey @elfeffe, can you share how you solved this issue? I'm facing it as well when I try to add the Alpine 3 script to a Livewire project.

**Reply by elfeffe** [reply]
I don't really know.
It suddenly dissapeared. 
I believe it was a problem with nested items, and in some point I have changed it and it begun to work.
Remember to clear the view cache completely

**Reply by christophrumpel** [reply]
Still thanks.
